app/services/dttp/import_user.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+module Dttp
+  class ImportUser
```

Comment:
I wonder if this is the best name. I mean, we are 'importing' the user but we're not fully importing them. We're sort of cacheing them so as they can be imported if we want to later. What would we call the service that takes this data and makes a register user from it? Maybe it's fine and we can worry about that when we do the next bit.

Reply:
I've binned it